_layouts/opengovsg-landing-page.html

```diff
@@ -257,7 +257,7 @@ <h4 class="padding--top--lg has-text-secondary"><b>UX Designer</b></h5>
 			</div>
 			<div class="row peopleops-row">
 				<div class="col">
-					<a href="http://go.gov.sg/ogp-people-ops" class="padding--top padding--bottom is-block">
+					<a href="http://go.gov.sg/ogp-jobs" class="padding--top padding--bottom is-block">
```

Comment:
https://  and there's also https://go.gov.sg/join-ogp on line 294 referencing the same url